deploy-service/common/src/main/java/com/pinterest/deployservice/db/DatabaseUtil.java

```diff
@@ -58,6 +59,7 @@ public static BasicDataSource createMysqlDataSource(String host, int port,
         "jdbc:mysql://%s:%d/deploy?connectTimeout=5000&socketTimeout=3000&characterEncoding=UTF-8"
             + "&connectionCollation=utf8_general_ci",
         host, port);
+    LOG.info("mtls is enabled --- host:{}, port:{}, user:{}, password:{}", host, port, user, passwd);
```

Comment:
Probably want to avoid logging password here

Reply:
> Probably want to avoid logging password here

sure